### PR TITLE
Added s3-bucket config to docker-compose

### PR DESCRIPTION
**Pull request by justinorringer**
### [RHCLOUD-21616](https://issues.redhat.com/browse/RHCLOUD-21616)
* `docker-compose up s3-bucket` automatically creates the `exports-bucket`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,12 +10,16 @@ For more information on integrating with the export service, see the [integratio
...
-You can start the database and minio using `podman-compose up db s3`. This will expose minio on `localhost:9099` 
-(use `minio` as the access key and `minioadmin` as the secret key)
+You can start the database, minio, and the api using `make run`. 
 
-Create a bucket called `exports-bucket` in minio, click **Manage** and set it's **Access Policy** to `public`.
+Ports now exposed:
+- public api on localhost:8000
+- metrics on localhost:9090
+- internal api on localhost:10010
+- minio on localhosst:9099
```
Comment (CodyWMitchell) [inline README.md:19]: typo in localhost here (sorry!)

File: docker-compose.yaml [hunk 2]
```diff
@@ -53,6 +53,18 @@ services:
       - MINIO_ROOT_PASSWORD=${AWS_SECRET_ACCESS_KEY-minioadmin}
     command: server --address 0.0.0.0:9099 --console-address 0.0.0.0:9990 /data
 
+  s3-bucket:
```
[the remark this answers is not in the document]
Comment (justinorringer) [reply docker-compose.yaml]: Changed :)

**Review by CodyWMitchell — APPROVED** [review]
Looks great!

**Review by dehort — APPROVED** [review]
LGTM!

**Pull request merged**